synfig-studio/src/gui/dialogs/dialog_guide.cpp

```diff
@@ -0,0 +1,261 @@
+/* === S Y N F I G ========================================================= */
+/*!	\file Dialog_Guide.cpp
+**	\brief Template Header
```

Comment:
Please add a simple description on \brief line. "Template Header" is the example of the... template file XD

Reply:
Oh looks like I changed it In the header but forgot the .cpp

Reply:
done